### ISSUE TITLE
When I train train_dual.py file, the val loss is 0 for each epoch

### ISSUE THREAD

**Issue by long-shen123** [WongKinYiu/yolov9#330]
When I train train_dual.py file, the val loss is 0 for each epoch. I see the previous issue replied and I want to know how to open the calculation of val loss when training train_dual.py file.

**Comment by Youho99** [comment]
You can't directly for the moment
But you Can use val_dual.py

**Comment by shaomei2023** [comment]
If I cannot calculate the validation loss, how to evaluate the performance of yolov9? how reliable is it to evaluate YOLOv9 compared to YOLOv8 and RT-DETR in this situation? Also, if the code for calculating validation loss at line 197 in val_dual.py is uncommented, i.e. loss += compute_loss(train_out, targets)[1]  # box, obj, cls, would the obtained results be reliable? Thank you!
